go/consensus/tendermint/seed/seed.go

```diff
@@ -136,14 +106,29 @@ func (srv *seedService) GetStatus(ctx context.Context) (*consensus.Status, error
 		Features: srv.SupportedFeatures(),
 	}
 
+	if srv.seedSrv == nil {
+		// Seed node not started yet.
+		return status, nil
+	}
+
 	// List of consensus peers.
-	tmpeers := srv.p2pSwitch.Peers().List()
-	peers := make([]string, 0, len(tmpeers))
-	for _, tmpeer := range tmpeers {
-		p := string(tmpeer.ID()) + "@" + tmpeer.RemoteAddr().String()
-		peers = append(peers, p)
+	status.NodePeers = []string{}
+	rpcenv := srv.seedSrv.RPCEnvironment()
```

Comment:
This appears to be shared between the full node and the seed node, perhaps this should be refactored out to a function.

Reply:
Good idea, I'll do that once I get the tests working :)